Assets/MixedRealityToolkit.Services/InputAnimation/Editor/InputRecordingWindow.cs

```diff
@@ -204,12 +207,12 @@ private void DrawPlaybackGUI()
             {
                 if (GUILayout.Button("Load ..."))
                 {
-                    string filepath = EditorUtility.OpenFilePanel(
+                    string filepath = EditorUtility.OpenFilePanelWithFilters(
                         "Select input animation file",
                         "",
-                        InputAnimationSerializationUtils.Extension);
+                        FileTypeFilters);
 
-                    LoadAnimation(filepath);
+                    TryLoadAnimation(filepath);
```

Comment:
Not sure how long the loading takes because of bug above. But may want to disable this button if it is truly async. The user could try to hit load again and re-select the file if it looks like nothing happened. 

Another consideration is to make the async call/task a one with progress and utilize EditorUtility.DisplayCancelableProgressBar()